server/src/main/scala/services/graphstages/Crunch.scala

```diff
@@ -165,6 +182,17 @@ object Crunch {
     withoutRemovalsWithUpdates
   }
 
+  def applyFlightsWithSplitsDiff(flightRemovals: Set[ArrivalKey], flightUpdates: Set[ApiFlightWithSplits], flights: Map[ArrivalKey, ApiFlightWithSplits]): Map[ArrivalKey, ApiFlightWithSplits] = {
+    val nowMillis = SDate.now().millisSinceEpoch
```

Comment:
should this now() be passed in for testing?

Reply:
good call